package.json

```
@@ -1,48 +1,55 @@
 {
-  "name": "slidev-theme-iodigital",
-  "version": "0.0.0",
-  "keywords": [
-    "slidev-theme",
-    "slidev"
-  ],
-  "engines": {
-    "node": ">=14.0.0",
-    "slidev": ">=0.19.3"
-  },
-  "scripts": {
-    "build": "slidev build example.md",
-    "dev": "slidev example.md --open",
-    "export": "slidev export example.md",
-    "screenshot": "slidev export example.md --format png",
-    "lint": "eslint --fix --ext .js,.ts,.vue ."
-  },
-  "dependencies": {
-    "@slidev/types": "^0.40.5",
-    "codemirror-theme-vars": "^0.1.2",
-    "prism-theme-vars": "^0.2.4",
-    "theme-vitesse": "^0.6.4"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^17.6.1",
-    "@commitlint/config-conventional": "^17.6.1",
-    "@iconify-json/mdi": "^1.1.50",
-    "@slidev/cli": "^0.40.5",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
-    "@typescript-eslint/parser": "^5.59.2",
-    "eslint": "^8.39.0",
-    "eslint-plugin-vue": "^9.11.0",
-    "husky": "^8.0.3",
-    "unplugin-auto-import": "^0.15.2"
-  },
-  "//": "Learn more: https://sli.dev/themes/write-a-theme.html",
-  "slidev": {
-    "colorSchema": "light",
-    "highlighter": "shiki",
-    "defaults": {
-      "favicon": "./logo/iO-logo_black.svg",
-      "transition": "slide-left",
-      "lineNumbers": true,
-      "hightlighter": "shiki"
+    "name": "@iodigital/slidev-theme-iodigital",
```

```suggestion
    "name": "slidev-theme-iodigital",
```

Package name must start with `slidev-theme-` https://sli.dev/themes/write-a-theme.html#conventions